controls/roles/manage-service/tasks/main.yml

```diff
@@ -37,6 +37,7 @@
       owner: "{{ stereum_service_configuration.user }}"
       group: "2000"
       mode: 0700
+    ignore_errors: yes  
```

Comment:
I think a better, safer, approach would be to:
- check if the path is a file or a directory
- set owner, group, mode for file (when file)
- set owner, group, mode for directory (when directory)

Reply:
yes, that would be best option, I'll have look

Reply:
solved using env & command, no need to mount "config file to file"